scripts/get_objf_and_derivs.py

```diff
@@ -249,6 +255,24 @@ def WriteDerivs():
         print("order{0}_D4 {1}".format(o, d4_deriv[o]), file=f)
     f.close()
 
+def CleanUp(need_model):
+    work_dir = args.work_dir
+    try:
+        if need_model == "false":
```

Comment:
I don't much like this way of doing things- people could easily create files in that directory and not know why they were deleted.  This script knows which files it created so it can easily name them one by one and delete them.  If this were run by mistake on someone's home directory, for instance, it could delete important files.